### PR TITLE
Create Fortune Teller chat Bot

### PR DESCRIPTION
**Pull request by cpz-crafts**

### CODE REVIEW

**Comment by cpz-crafts** [comment]
https://amazing-longma-da9b65.netlify.app/

File: code/script.js [hunk 1]
```diff
@@ -31,21 +65,252 @@ const showMessage = (message, sender) => {
...
+
+}
+ // function that pick a random fortune string 
+const getFortune = () => {
+  const selectedIndex = Math.floor(Math.random() * fortunes.length);
+  return fortunes[selectedIndex];
 }
 
-// Set up your eventlisteners here
+// function that display the video of choosing your zodiac sign
```
Comment (code-and-cats) [inline code/script.js:272]: Very nice use of conditional statements!

File: code/script.js [hunk 2]
```diff
@@ -31,21 +65,252 @@ const showMessage = (message, sender) => {
...
+  <button id="Astrology" type="submit">Astrology</button> 
+  <button id="Fortune" type="submit">Fortune</button>
+  `
+  //choices for the user                              
+  document.getElementById('Shop').addEventListener('click', () => {
+    showMessage("Please show me the shop", 'user')
+    setTimeout(() => answerTwo('Shop', 'user')), 1000
+  })
+  document.getElementById('Astrology').addEventListener('click', () => {
+    showMessage("I would love to know about Atrology.", 'user')
```
Comment (code-and-cats) [inline code/script.js:102]: just a small typo here: "Astrology"

File: code/script.js [hunk 3]
```diff
@@ -31,21 +65,252 @@ const showMessage = (message, sender) => {
...
+}
+
+const greetingAnswer = (event) => {
+  event.preventDefault();   // for not refresh website
+  const userName = nameInput.value
+  showMessage(`My name is ${userName}`, 'user');
+  nameInput.value = ''          // setting the users answer 
+  showMessage(`Hello! ${userName} Choose a topic you would like to know about?`, 'bot');
+  setTimeout(() => questionTwo(userName), 1000) // time for the funtion to execute 
+}
```
Comment (code-and-cats) [inline code/script.js:83]: I like the clear explanations here, makes the code easy to follow

File: code/style.css [hunk 4]
```diff
@@ -124,16 +154,61 @@ label {
...
+.dropDown {
+
+  background: rgba(255, 255, 255, 0.41);
+  border-radius: 16px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(12.9px);
+  -webkit-backdrop-filter: blur(12.9px);
+  border: 1px solid rgba(255, 255, 255, 0.32);
+
   color: white;
```
Comment (code-and-cats) [inline code/style.css:208]: For me, the dropDown worked as intended, but I couldn't see the number on the options. Might changing color solve that?

File: code/script.js [hunk 5]
```diff
@@ -31,21 +65,252 @@ const showMessage = (message, sender) => {
...
+  } else if ((month == 1 && day >= 20) || (month == 2 && day <= 18)) {
+    return "Aquarius: Aquarius is the eleventh sign of the zodiac, and it is associated with independence, originality, and humanitarianism. People born under this sign are known for their innovative ideas, progressive attitudes, and the ability to see the bigger picture. They are often described as eccentric, non-conformist, and quirky, and they value freedom and individuality above all else.";
+  } else if ((month == 2 && day >= 19) || (month == 3 && day <= 20)) {
+    return "Pisces: Pisces is the twelfth and final sign of the zodiac, and it is associated with empathy, intuition, and creativity. People born under this sign are known for their compassionate and empathetic nature, and they have a strong connection to their emotions and the emotions of others.Pisces people are often drawn to the arts, and they have a natural talent for music, dance, and other creative pursuits.";
+  } else {
```
Comment (code-and-cats) [inline code/script.js:299]: nice adding a fallback!

**Review by code-and-cats — APPROVED** [review]
Great job! 🔮 The code is easy to follow and well structured. I liked the use of conditionals in several different places. Also very cool to use Math.random for the fortunes! As far as I can tell it ticks all the requirements up to black level.
One suggestion might be to add a reset button or something after the final question, if the user wants to go again, but it works nice without it as well.

**Pull request closed without merging**